style.css

```diff
@@ -0,0 +1,296 @@
+* {
+  box-sizing: border-box;
+  margin: 0;
+  padding: 0;
+}
+
+body {
+  font-family: Poppins, sans-serif;
+  width: 100%;
+  height: 100vh;
+  margin: 0;
+  padding: 0;
+}
+
+/* Header */
+header {
+  display: flex;
+  max-width: 500px;
+  margin: 0 auto;
+  justify-content: space-around;
+  align-items: center;
+}
+
+header a {
+  color: #6070ff;
+  font-size: 1.5rem;
+  padding: 0 40px;
+  text-decoration: none;
+  font-weight: bold;
+}
+
+nav {
+  margin: 0 auto;
+}
+
+/* section */
+
+.headline-section {
+  color: #091e42;
+  max-width: 500px;
+  margin: 0 auto;
+  background-image: url(https://slack-imgs.com/?c=1&o1=ro&url=https%3A%2F%2Fraw.githubusercontent.com%2Fhammaazarok%2Fportfolio%2Fmaster%2Fimg%2Fbg.png);
+  background-repeat: no-repeat;
+  background-position: left right;
+  padding: 40px 30px;
+  text-align: right left;
+  background-color: #fff;
+  border: 2px solid #fff;
+  border-bottom-left-radius: 40px 20px;
+}
+
+.headline-section h1 {
+  margin: 10px;
+  color: #091e42;
+  font-weight: 800;
+  font-size: 2rem;
+  width: 60%;
+}
+
+.headline-section p {
+  color: #344563;
+  width: 60%;
+  margin: 10px;
+}
+
+.headline-section ul {
+  padding: 10px;
+  color: #505f79;
+  width: 60%;
+  display: flex;
+  list-style-type: none;
+}
+
+.headline-section ul li {
+  padding: 5px;
+  margin-right: 10px;
+}
+
+.text {
+  text-decoration: none;
+  font-size: 16px;
+  margin-left: 10px;
+  color: #7f8cff;
+}
+
+.btn {
+  border: none;
+  background-color: #fff;
+  color: #6070ff;
+  font-size: 30px;
+  margin-left: 70px;
+}
+
+.work-section {
+  color: #091e42;
+  max-width: 500px;
+  margin: 0 auto;
+  padding: 40px 30px;
+  display: grid;
+  grid-template-columns: 1fr;
+  grid-gap: 50px;
+  text-align: right left;
+}
+
+.workd-card h1 {
+  color: #172b4d;
+  font-weight: bold;
+  font-size: 2rem;
+  width: 60%;
+  margin: 10px;
+  margin-top: 10px;
+  text-align: right left;
+}
+
+.work-section p {
+  color: #344563;
+  width: 70%;
+  margin: 10px;
+}
+
+.big-button {
+  padding: 10px;
+  border-radius: 8px;
+  color: #396df2;
+  text-align: center;
+  background-color: #fff;
+  border: 1px solid #6070ff;
+  font-weight: bolder;
+  width: 30%;
+  margin-left: 15px;
+  margin-bottom: -20px;
+  font-size: 17px;
+  letter-spacing: 3%;
+}
```

Comment:
- [ ] kindly align the button to the left to look similar to the Figma template. 👇🏽 

| **Figma template** | **Your template** |
|---|---|
| ![image](https://user-images.githubusercontent.com/58642949/192791371-154e350b-809b-49e0-9db3-329681f50007.png) | ![image](https://user-images.githubusercontent.com/58642949/192791516-950492cc-f829-40f9-ad91-344a6e5f154e.png) |